artemis/modules/bruter.py

```diff
@@ -54,6 +56,32 @@ class BruterResult:
     checked_paths: List[str]
 
 
+class BruterConfiguration(ModuleConfiguration):
+    """Configuration for the bruter module."""
+
+    def __init__(self, enabled: bool = True, max_attempts: int = 1000) -> None:
+        super().__init__(enabled=enabled)
+        self.max_attempts = max_attempts
+
+    def serialize(self) -> Dict[str, Any]:
+        result = super().serialize()
+        result["max_attempts"] = self.max_attempts
```

Comment:
what is max_attempts? I don't see that implemented

If you want to provide something other than nuclei as well, whast do you think about including warnings in mail_DNS_scanner reporter?

Reply:
I don't think this one is resolved ;)